### ISSUE TITLE
Does not work anymore?

### ISSUE THREAD

**Issue by Shellfishgene** [robertnavca/mobileR#1]
Hi,
this would be very useful but I can't get it to work with the example given, it returns `Fehler in 1:number_of_pages : NA/NaN Argument`. Maybe the site changed formatting?

**Comment by lonione** [comment]
Same error here - I suspect it's because the script gets no data at all. Not sure how to debug though. Did you find a way how to make it work?

**Comment by jenslaufer** [comment]
same here

**Comment by LukasPeterWIN** [comment]
Hi, I get the error in open.connection(x, "RB") : HTTP error 403. Is this still being worked on?

**Comment by lonione** [comment]
I strongly suspect the issue here is that Mobile.de is now actively preventing this kind of programmatic access - instead of the expected page with data a "robot" check page is displayed (i.e. Are you a robot?) with captcha... the script obviously can't read this and throws an error.

**Comment by LukasPeterWIN** [comment]
Yes that is probably true. But isn't the API public? Doesn't that help? I dont know much about that stuff..

**Comment by lonione** [comment]
AFAIK the API is a paid feature, and this script does not use it. It called the standard "human" web interface and scraped it for data.

**Comment by julesbld** [comment]
Following. I'm having the same error. Any solution found? Cheers.

**Comment by wojciechniemczyk** [comment]
Error in open.connection(x, "rb") : HTTP error 403.
